### DISCUSSION TITLE
[WARN]  grpc-py: plugin failed to exit gracefully

### DISCUSSION THREAD

**Discussion by pyselehot** [httprunner/httprunner#1388] [2 - 答疑（Q&A）]
==================== response ====================
Connected via TLSv1.2
HTTP/1.1 200 OK
Content-Length: 194
Connection: keep-alive
Content-Type: application/json; encoding=utf-8
Date: Sat, 25 Jun 2022 15:42:26 GMT

{"access_token":"58_uq3JOJp-shz0dGRKF1ZWPP3pxJyTfckWfQlc9YhvLXc_2NwRJbk6W9k84r6pq3kovXykPRxvG-nASjViKhRBMDFnylKaGwJjtDo58UjlWGqkKS2D8JJUDhbuknOkS2BcfpyuEHHarxHOD4WIOCGeADAKYC","expires_in":7200}
--------------------------------------------------
11:42PM INF extract value from=body.access_token value=58_uq3JOJp-shz0dGRKF1ZWPP3pxJyTfckWfQlc9YhvLXc_2NwRJbk6W9k84r6pq3kovXykPRxvG-nASjViKhRBMDFnylKaGwJjtDo58UjlWGqkKS2D8JJUDhbuknOkS2BcfpyuEHHarxHOD4WIOCGeADAKYC
11:42PM INF set variable value=58_uq3JOJp-shz0dGRKF1ZWPP3pxJyTfckWfQlc9YhvLXc_2NwRJbk6W9k84r6pq3kovXykPRxvG-nASjViKhRBMDFnylKaGwJjtDo58UjlWGqkKS2D8JJUDhbuknOkS2BcfpyuEHHarxHOD4WIOCGeADAKYC variable=access_token
11:42PM INF validate status_code assertMethod=eq checkExpr=status_code checkValue=200 checkValueType=int64 expectValue=200 expectValueType=int64 result=true
11:42PM INF validate body.expires_in assertMethod=eq checkExpr=body.expires_in checkValue=7200 checkValueType=int64 expectValue=7200 expectValueType=int64 result=true
11:42PM INF validate body assertMethod=contains checkExpr=body checkValue={"access_token":"58_uq3JOJp-shz0dGRKF1ZWPP3pxJyTfckWfQlc9YhvLXc_2NwRJbk6W9k84r6pq3kovXykPRxvG-nASjViKhRBMDFnylKaGwJjtDo58UjlWGqkKS2D8JJUDhbuknOkS2BcfpyuEHHarxHOD4WIOCGeADAKYC","expires_in":7200} checkValueType="map[string]interface {}" expectValue=access_token expectValueType=string result=true
11:42PM INF run step end exportVars={"access_token":"58_uq3JOJp-shz0dGRKF1ZWPP3pxJyTfckWfQlc9YhvLXc_2NwRJbk6W9k84r6pq3kovXykPRxvG-nASjViKhRBMDFnylKaGwJjtDo58UjlWGqkKS2D8JJUDhbuknOkS2BcfpyuEHHarxHOD4WIOCGeADAKYC"} step=get_token success=true type=request
11:42PM INF run testcase end testcase="获取鉴权码access token接口"
11:42PM INF quit hashicorp plugin process
2022-06-25T23:42:28.416+0800 [WARN]  grpc-py: plugin failed to exit gracefully

进程已结束,退出代码0

**Comment by HorsonA** [comment]
我也遇到了这个问题，请问解决了吗？

**Reply by Biuuuuu-19** [reply]
hrp boom 跑完性能，我也遇到 [WARN]  grpc-py: plugin failed to exit gracefully

**Comment by kvchs** [comment]
遇到这个问题了，求解
